sidebar_translations.json

```
@@ -62,17 +62,19 @@
   "develop.loom.classpath_groups": "Classpath Groups",
   "develop.loom.tasks": "Other Tasks",
   "develop.misc": "Miscellaneous Pages",
+  "develop.misc.automatic_testing": "Automated Testing",
   "develop.misc.codecs": "Codecs",
   "develop.misc.data_attachments": "Data Attachments",
   "develop.misc.events": "Events",
   "develop.misc.networking": "Networking",
   "develop.misc.saved_data": "Saved Data",
   "develop.misc.text_and_translations": "Text and Translations",
   "develop.misc.debugging": "Debugging Mods",
-  "develop.misc.migrating_mappings": "Migrating Mappings",
-  "develop.misc.migrating_mappings.loom": "Loom (Gradle)",
-  "develop.misc.migrating_mappings.ravel": "Ravel (IntelliJ IDEA)",
-  "develop.misc.automatic_testing": "Automated Testing",
+  "develop.porting": "Porting",
+  "develop.porting.introduction": "Introduction",
+  "develop.porting.migrating_mappings": "Migrating Mappings",
```

```suggestion
  "develop.porting.migrating_mappings": "Mappings",
```

Because it is already "Porting > Mappings"

---

With this change, I would end up reading it as just Mappings, which is an inaccurate title. This also better matches the other porting articles.

---

That was my intention: iirc Mappings also explains what mappings are

EDIT: I checked it; apparently it does but it starts with Migration instructions, maybe that should be reversed?

---

I believe we had originally planned to expand Mappings into its own article as it was on the wiki, but its less relevant now that mappings are a thing of the past. The Migrating Mappings page has enough context on what mappings _are_ in order to understand what remapping is. Perhaps a future PR can rework this article if this should no longer bee the case.